safe/gui/tools/peta_bencana_dialog.py

```diff
@@ -243,9 +243,19 @@ def accept(self):
 
         self.copy_keywords(output_base_file_path)
         layer = self.add_flooded_field(output_base_file_path)
-        # add the layer to the map
-        registry = QgsMapLayerRegistry.instance()
-        registry.addMapLayer(layer)
+
+        # check if the layer has feature or not
+        if layer.featureCount() <= 0:
+            message = 'There are no floods data available at this time.'
```

Comment:
at this time in this area ? I don't know if the tool takes care about extent ?
I just checked in QGIS, we don't know if it takes QGIS extent, all Jakarta extent (as it was call Peta**Jakarta**)
Maybe we should check with @lucernae or @timlinux , it something we should fix IMHO.

BTW, I think it's better to apply the tr() straight on the string, on this line.